pages/steps/permutations-utils.tsx

```diff
@@ -0,0 +1,774 @@
+    status: 'error',
+    statusIconAriaLabel: 'error',
+  },
+];
+
+export const failedStepsWithRetryTextInteractive: ReadonlyArray<StepsProps.Step> = [
+  {
+    header: (
+      <span style={{ margin: '0' }}>
+        Listed EC2 instances:{' '}
+        <Popover
+          header={'EC2 Instance IDs'}
+          content={
+            <span style={{ marginBlock: 0, fontSize: '12px' }}>
+              <ul>
+                <li>ec2InstanceID1</li>
+                <li>ec2InstanceID2</li>
+                <li>ec2InstanceID3</li>
+                <li>ec2InstanceID4</li>
+              </ul>
+            </span>
+          }
+          position={'bottom'}
+        >
+          4 items
+        </Popover>
+      </span>
+    ),
+    status: 'success',
+    statusIconAriaLabel: 'success',
+  },
+  {
+    header: <span style={{ margin: '0' }}>Could not fetch security groups</span>,
+    details: (
+      <span style={{ margin: '0' }}>
+        The request expired due to a server issue. <Link href="#">Retry</Link>
+      </span>
+    ),
+    status: 'error',
+    statusIconAriaLabel: 'error',
+  },
+];
+
+export const failedStepsWithRetryButtonInteractive: ReadonlyArray<StepsProps.Step> = [
+  {
+    header: (
+      <span style={{ margin: '0' }}>
+        Listed EC2 instances:{' '}
+        <Popover
+          header={'EC2 Instance IDs'}
+          content={
+            <span style={{ marginBlock: 0, fontSize: '12px' }}>
+              <ul>
+                <li>ec2InstanceID1</li>
+                <li>ec2InstanceID2</li>
+                <li>ec2InstanceID3</li>
+                <li>ec2InstanceID4</li>
+              </ul>
+            </span>
+          }
+          position={'bottom'}
+        >
+          4 items
+        </Popover>
+      </span>
+    ),
+    status: 'success',
+    statusIconAriaLabel: 'success',
+  },
+  {
+    header: <span style={{ margin: '0' }}>Could not fetch security groups</span>,
+    details: (
+      <>
+        <span style={{ margin: '0' }}>The request expired due to a server issue.</span>
+        <Button ariaLabel="Retry" iconName="refresh">
+          Retry
+        </Button>
+      </>
+    ),
+    status: 'error',
+    statusIconAriaLabel: 'error',
+  },
+];
+
+const changesetStepsInteractive: ReadonlyArray<StepsProps.Step> = [
+  {
+    status: 'success',
+    statusIconAriaLabel: 'success',
+    header: (
+      <span style={{ margin: '0' }}>
+        Create S3 <Link href="#">bucket-12</Link>
+      </span>
+    ),
+  },
+  {
+    status: 'success',
+    statusIconAriaLabel: 'success',
+    header: (
+      <span style={{ margin: '0' }}>
+        Create CloudWatch <Link href="#">alarm</Link>
+      </span>
+    ),
+  },
+  {
+    status: 'loading',
+    statusIconAriaLabel: 'loading',
+    header: (
+      <span style={{ margin: '0' }}>
+        Update <Link href="#">Lambda-1</Link>
+      </span>
+    ),
+  },
+];
+
+export const stepsPermutationsInteractive = createPermutations<StepsProps>([
```

Comment:
This export (and the steps within it) doesn't seem to be used

Reply:
You were right! But now it's used again :) sorry